vars/spinInfra.groovy

```diff
@@ -6,20 +6,18 @@ import uk.gov.hmcts.contino.TerraformTagMap
 import uk.gov.hmcts.contino.MetricsPublisher
 import uk.gov.hmcts.pipeline.AKSSubscriptions
 import uk.gov.hmcts.contino.RepositoryUrl
-
-def call(productName, environment, tfPlanOnly, subscription) {
-  call(productName, null, environment, tfPlanOnly, subscription)
-}
-
-def call(product, component, environment, tfPlanOnly, subscription) {
-  call(product, component, environment, tfPlanOnly, subscription, "")
-}
-
-def call(product, component, environment, tfPlanOnly, subscription, deploymentTarget) {
```

Comment:
removing all these is a breaking change, you should keep all existing ones adapting them to pass parameters to the new method